homeassistant/helpers/data_entry_flow.py

```diff
@@ -86,6 +87,13 @@ async def post(self, request: web.Request, data: dict[str, Any]) -> web.Response
 
         return self.json(result)
 
+    def get_context(self, data: dict[str, Any]) -> dict[str, Any]:
+        """Return context."""
+        return {
+            "source": config_entries.SOURCE_USER,
+            "show_advanced_options": data["show_advanced_options"],
+        }
```

Comment:
I think we should add the source, which seems to be config flow specific, in `ConfigManagerFlowIndexView.get_context`
```suggestion
        return {
            "show_advanced_options": data["show_advanced_options"],
        }
```